requirements-hpu.txt

```diff
@@ -9,4 +9,5 @@ tabulate
 setuptools>=61
 setuptools-scm>=8
 vllm-hpu-extension @ git+https://github.com/HabanaAI/vllm-hpu-extension.git@250622e
+neural-compressor @ git+https://github.com/intel/neural-compressor.git@b196432 --global-option="--upgrade"
```

Comment:
The global-option seems to be causing issues with pip install called in cpu_test. Is this needed since a specific version is provided after @?

Reply:
@michalkuligowski , do you mean the requirements-hpu.txt will also be used in CPU test?

The reason for using --upgrade is because released synapse docker has a preinstalled Neural-compressor, if not adding '--upgrade', 'pip install -r requirements-hpu.txt' will not update to this new version.